### ISSUE TITLE
Feature Request: pasteboard function

### ISSUE THREAD

**Issue by whytte** [Crypto-Notepad/Crypto-Notepad#53]
Keep the Crypto-Notepad open in task bar
whatever copied from web searches past onto a single text file continuously

**Comment by Sigmanor** [comment]
It is not entirely clear what you mean, please describe in more details.

**Comment by whytte** [comment]
Basically it is a pasteboard function , all copied text one by one  from different sources go to a open single/same  text file continuously 

As a researcher  I do internet surfing a lot and copy & paste many articles using Firefox browser
While I see  many articles of interest, I copy them and paste into text editor
which means, copy from web , paste into editor one by one and so on

Instead of copy and paste one by one
I'd like to keep the text editor like "Crypto-Notepad" open/active, waiting in the task bar
and whatever I copy from web sources  I'd like them to be  accumulated /pasted automatically, continuously on a single open text file of an editor like "Crypto-Notepad" for me.
Basically it is a pasteboard function , all copied text one by one  from the windows clipboard go to a open single/same  text file continuously

**Comment by whytte** [comment]
similar Pasteboard (capture Clipboard text) example in another text editor:

> https://www.notetab.com/comparison-chart
info:
Capture Text Copied to the Clipboard.
Copy and pasting is another feature that is constantly used in a text editor. Think about this. How do you collect data from various sources like web pages and other documents? Here’s how most people do it:
    Click on the window that has the data you need.
    Select and copy the data to the clipboard.
    Click on your text editor.
    Paste the data.
    Repeat steps 1 to 4 …
    On the other hand NoteTab has a feature called the Paste Board. Once you turn it on, any text you copy to the clipboard is automatically pasted into your chosen document.  ``

**Comment by Sigmanor** [comment]
okay  I will look into it later, and give you an answer

**Comment by Sigmanor** [comment]
Well look like I can add this function to the Crypto Notepad, although for me personally, this function will not be useful, but it sounds useful for such program as Crypto Notepad.

**Comment by whytte** [comment]
Thank you  Sigmanor.
Appreciate your prompt and timely work very much
all the best

**Comment by Sigmanor** [comment]
**Preview:**
![](https://i.imgur.com/nHQCoEc.gif)

**Comment by whytte** [comment]
![thank you](https://user-images.githubusercontent.com/71485634/114304893-4a021600-9ade-11eb-9cfc-3c4549f76650.jpg)

Thank you for the wonderful news today Alexander.
I really appreciate your kind approach to implement a feature I really wanted.
I hope more users will be able to utilize the pasteboard function.
 Started using it right now.
 all the best
 
 please do let me know if I could be any help
